pkg/utils/createorupdate.go

```diff
@@ -46,7 +46,7 @@ func CreateOrUpdateClusterRole(clientSet clientset.Interface, clusterRole *rbacv
 			// Potentially retried
 			_, err = clientSet.RbacV1().ClusterRoles().Update(clusterRole)
 			if err != nil {
-				return fmt.Errorf("failed to update pre-existing cluster role %s : %v", clusterRole.Name, err)
+				return err
```

Comment:
you can remove the if check all-together and simply use "return err"

Reply:
> you can remove the if check all-together and simply use "return err"

Will do. I was considering checking for conflict error, but late realized we're logging error in caller.